lib/Controller/ContextsController.php

```diff
@@ -0,0 +1,66 @@
+<?php
```

Comment:
Since other controller files use singular names, maybe rename this file to `ContextController.php` for consistency?